vercel.json

```diff
@@ -4,5 +4,16 @@
 			"source": "/login/:path*",
 			"destination": "/:path*"
 		}
-	]
+	],
+  "headers": [
+    {
+      "source": "/api/(.*)",
+      "headers": [
+        { "key": "Access-Control-Allow-Credentials", "value": "true" },
+        { "key": "Access-Control-Allow-Origin", "value": "*" },
+        { "key": "Access-Control-Allow-Methods", "value": "GET" },
+        { "key": "Access-Control-Allow-Headers", "value": "X-CSRF-Token, X-Requested-With, Accept, Accept-Version, Content-Length, Content-MD5, Content-Type, Date, X-Api-Version" }
+      ]
+    }
+  ]
```

Comment:
@omercnet what do you think?
We need the Vercel functions to be reachable locally too, and by customer who run their own auth hosting